src/frontend/src/components/ScrollToTop/ScrollToTop.jsx

```diff
@@ -58,7 +58,6 @@ export default function BackToTop(props) {
   const classes = useStyles();
   return (
     <>
-      {/* TODO: remove this part of issue: #1206: delete the line and the "anchor" class */}
```

Comment:
Both lines 61 and 62 had to be removed, I updated the issue to be more clear, sorry!